db/sqlinterfaces.c

```diff
@@ -3554,7 +3554,13 @@ static void setup_reqlog_new_sql(struct sqlthdstate *thd,
         snprintf(info_nvreplays, sizeof(info_nvreplays), "vreplays=%d ",
                  clnt->verify_retries);
 
-    thrman_wheref(thd->thr_self, "%ssql: %s", info_nvreplays, clnt->sql);
+    if (clnt->sql_query && clnt->sql_query->client_info) {
+        thrman_wheref(thd->thr_self, "%s pid: %d host_id: %d sql: %s",
+                      info_nvreplays, clnt->sql_query->client_info->pid,
+                      clnt->sql_query->client_info->host_id, clnt->sql);
+    } else {
+        thrman_wheref(thd->thr_self, "%s sql: %s", info_nvreplays, clnt->sql);
```

Comment:
A space is already appended to `info_nvreplays`. You might want to either remove the space in the format string here, or remove the trailing space in the snprintf above.

Reply:
done